docs/zh/client/java-sdk-user-guide.md

```
@@ -1220,3 +1311,263 @@
+          registry.metricFamilySamples());
+      return writer.toString();
+    } catch (IOException e) {
+      logger.error("Write metrics to Prometheus format failed", e);
+      return "";
+    }
+  }
+```
+
+至此，已经将Client的指标数据接入Prometheus。
+
+完整代码:[PrometheusApolloClientMetricsExporter.java]https://github.com/apolloconfig/apollo-java/main/master/apollo-plugin/apollo-plugin-client-prometheus/src/main/java/com/ctrip/framework/apollo/monitor/internal/exporter/impl/PrometheusApolloClientMetricsExporter.java
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)